From bffc569d2cdce73ac8d998e4328cccb1393fb272 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Thu, 16 May 2019 10:44:22 +0100
Subject: [PATCH] TOOLS-2230 sdc-amon build fails due to missing zutil
 configure Reviewed by: Chris Burroughs <chris.burroughs@joyent.com> Approved
 by: Chris Burroughs <chris.burroughs@joyent.com>

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 985b52a..5255d77 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -85,7 +85,7 @@ endif
 # an agent installation or uninstallation, so we provide a magic environment
 # varible to disable them here.
 #
-NPM_ENV = SDC_AGENT_SKIP_LIFECYCLE=yes MAKE=$(MAKE)
+NPM_ENV = npm_config_cache=$(TOP)/$(BUILD)/.npm SDC_AGENT_SKIP_LIFECYCLE=yes MAKE=$(MAKE)
 
 NODE_DEV := ./node_modules/.bin/node-dev
 TAP := ./node_modules/.bin/tap
-- 
2.21.0

